File: Modul 3/Soal 2/PRAK302-2410817210019-AchmadReihanAlfaiz.c
#include <stdio.h>

int main()
{
     int userInput;
     scanf("%d", &userInput);

     char nilaiHuruf = (userInput >= 80) ? 'A' : (userInput >= 70) ? 'B'
                                             : (userInput >= 60)   ? 'C'
                                             : (userInput >= 50)   ? 'D'
                                                                   : 'E';

     printf("\nOutput: \n%c", nilaiHuruf);

     return 0;
}

File: Modul 3/Soal 2/PRAK302-2410817210019-AchmadReihanAlfaiz.py
userInput = int(input())

nilaiHuruf = (
    "A"
    if userInput >= 80
    else "B"
    if userInput >= 70
    else "C"
    if userInput >= 60
    else "D"
    if userInput >= 50
    else "E"
)

print(f"\nOutput: \n{nilaiHuruf}")
